package.json

```diff
@@ -25,6 +25,7 @@
     "addon"
   ],
   "scripts": {
+    "doc": "documentation build src/webextension/** -f md -o docs/api.md",
```

Comment:
Should we have a script for actually running MkDocs here? (Bonus if we run it and deploy our docs to Github in Travis CI.)

Reply:
Yup, sorry I didn't get that far. I wanted to make sure we wer eon the right track. I've got this further along over in lockbox-datastore.

I will finish up over there (assuming @linuxwolf is cool with the approach too) and then backport the final bits over here...